packages/abi-coder/src/FunctionFragment.ts

```diff
@@ -28,9 +28,9 @@ export class FunctionFragment<
   readonly jsonFn: JsonAbiFunction;
   readonly attributes: readonly JsonAbiFunctionAttribute[];
```

Comment:
Should these reference the new ABI?

Reply:
Fixed it in https://github.com/FuelLabs/fuels-ts/pull/2862/commits/5a1776f1ec4b612f648ee194f2564b86b518fb14